doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1247,23 +1248,24 @@ Here are the syntax elements used by the various notation commands.
              due to legacy notation in the Coq standard library.
              It can be turned on with the ``-w disj-pattern-notation`` flag.
 
+.. note::
+
+   The entry ``strict ident`` parses identifiers as in grammar entry
+   :n:`@ident`. The entry ``name`` parses :n:`@name`, that is
+   identifiers and :g:`_`. The entry ``global`` parses :n:`@qualid`
+   (and thus, as a special case, :n:`@ident`). The entry ``bigint``
+   parses arbitrary large integer numbers as in :n:`@bigint`.
```

Comment:
```suggestion
   ``global``
     Parses :n:`@qualid` (and thus, as a special case, :n:`@ident`).
     
   ``bigint``
      Parses arbitrary large integer numbers as in :n:`@bigint`.

   ``strict ident``
     Parses identifiers as in grammar entry
     :n:`@ident`. The entry ``name`` parses :n:`@name`, which allows
     both :n:`@ident`\s and :g:`_`.
```

The order of these items should be the same as in the grammar.  And move this above all the notes, around line 1217

Reply:
OK, adding then a paragraph for `name` too I guess.